arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1958,55 +1962,25 @@
       "properties": {
         "provisioningState": {
           "type": "string",
+          "readOnly": true,
           "description": "Provisioning state of the namespace."
         },
-        "status": {
```

Comment:
removing fields is breaking change. Can you elaborate more whether it was added by mistake in earlier or is it a service change.